pkg/controller/machine/machine_controller.go

```diff
@@ -18,18 +18,25 @@ package machine
 
 import (
 	"context"
+	"fmt"
 	"path"
 	"sync"
+	"time"
 
+	"github.com/go-log/log/info"
```

Comment:
We shouldn't be using another logging library, CAPI is using `klog`, or where applicabe, `klogr`

Reply:
This actually isn't a logging library, it's an interface library.

Reply:
As for interface, we usually use https://github.com/go-logr/logr

Reply:
Okay.  I can see about refactoring to include the kubectl library for drain and then we'll see if this is still necessary.

Reply:
This seems impractical if we don't want to vendor kubernetes/kubernetes.